app/assets/stylesheets/application.css

```diff
@@ -12,5 +12,225 @@
  *
  *= require_tree .
  *= require_self
- *= require foundation_and_overrides
```

Comment:
oops, sorry I just copied and pasted the code from application.css that I pushed to submission branch didn't noticed you changed it.